outputs.tf

```
@@ -28,31 +38,21 @@ output "ingress" {
   }
 }
 
-output "product_urls" {
-  description = "URLs to access the deployed Atlassian products"
-
-  value = {
-    bamboo = module.bamboo.product_domain_name
-  }
-}
-
-output "database" {
+output "bamboo-database" {
```

```suggestion
output "bamboo_database" {
```

I think we are using `_` underscore everywhere.